File: src/Omnia.ProcessManagement.Web/client/fx/models/data/shapetemplates/ShapeTemplateFreeformSettings.ts
import { ShapeTemplateSettings } from './ShapeTemplateSettings';
import { FabricShapeData } from '../../../processshape/fabricshape/FabricShapeData';
import { ShapeTemplateType } from '../enums/Enums';

export interface ShapeTemplateFreeformSettings extends ShapeTemplateSettings {
    type: ShapeTemplateType.FreeformShape,
    nodes: FabricShapeData[];
}

File: src/Omnia.ProcessManagement.Models/ShapeTemplates/ShapeTemplateFreeformSettings.cs
using Omnia.ProcessManagement.Models.FabricShape;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ShapeTemplates
{
    public class ShapeTemplateFreeformSettings : ShapeTemplateSettings
    {
        public List<FabricShapeData> Nodes { get; set; }
    }
}
